src/CoverageTransformer.js

```diff
@@ -123,7 +123,7 @@ export default class CoverageTransformer {
 		if (sourceMap.sourcesContent) {
 			origSourceFilename = rawSourceMap.sources[0];
 
-			if (origSourceFilename && path.extname(origSourceFilename) !== '') {
+			if (origSourceFilename && path.extname(origSourceFilename) !== '' && rawSourceMap.sources.length === 1) {
 				origFileName = rawSourceMap.file;
```

Comment:
@dylans @mattlewis92 My only concern is that if there were multiple sources this code would not process sources at all in that scenario. However, this code does address the original issue and puts it back more-or-less at parity with prior to the change that caused that issue in the first place (in terms of this particular section of code). I am OK with landing this and then we can assess whether or not we need to look at it further. I will merge it right now.

Thanks for your work @mattlewis92 !